pkg/cephfs/nodeserver.go

```diff
@@ -241,7 +265,3 @@ func (ns *NodeServer) NodeGetCapabilities(ctx context.Context, req *csi.NodeGetC
 		},
 	}, nil
 }
-
-func (ns *NodeServer) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest) (*csi.NodeGetInfoResponse, error) {
```

Comment:
we still need this

Reply:
we don't need this one as the same function is implemented in CSI common https://github.com/kubernetes-csi/drivers/blob/master/pkg/csi-common/nodeserver-default.go#L39

Reply:
but we have to call the default implementation in our node server

Reply:
@rootfs everything worked fine in during my testing let me check once again

Reply:
@rootfs  default node server is getting called

```
I0129 13:59:10.161483       1 nodeserver-default.go:40] Using default NodeGetInfo
I0129 13:59:10.161504       1 utils.go:103] GRPC response: {"node_id":"kube2"}
```